--- origsrc/icu/source/tools/genrb/derb.c	2013-03-15 17:12:04.000000000 -0500
+++ src/icu/source/tools/genrb/derb.c	2013-04-15 20:25:29.738717600 -0500
@@ -35,9 +35,11 @@
 #include <fcntl.h>
 #define USE_FILENO_BINARY_MODE 1
 /* Windows likes to rename Unix-like functions */
+#if U_PLATFORM != U_PF_CYGWIN
 #ifndef fileno
 #define fileno _fileno
 #endif
+#endif
 #ifndef setmode
 #define setmode _setmode
 #endif
--- origsrc/icu/source/tools/toolutil/udbgutil.cpp	2013-03-15 17:12:08.000000000 -0500
+++ src/icu/source/tools/toolutil/udbgutil.cpp	2013-04-15 20:18:31.180290700 -0500
@@ -339,8 +339,10 @@ int32_t udbg_enumByName(UDebugEnumType t
  */
 U_CAPI const char *udbg_getPlatform(void)
 {
-#if U_PLATFORM_HAS_WIN32_API
+#if U_PLATFORM_USES_ONLY_WIN32_API
     return "Windows";
+#elif U_PLATFORM == U_PF_CYGWIN
+    return "Cygwin";
 #elif U_PLATFORM == U_PF_UNKNOWN
     return "unknown";
 #elif U_PLATFORM == U_PF_DARWIN
